task/rhacs-m2m-authenticate/4.4.2/samples/pipelinerun.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: rox-pipelinerun
+spec:
+  serviceAccountName: build-bot
+  pipelineRef:
+    name: rox-pipeline
+  params:
+  - name: deployment-name
+    value: pipelines-vote-api
+  - name: deployment
+    value: k8s/deployment.yaml
+  - name: git-url
+    value: https://github.com/openshift/pipelines-vote-api.git
+  - name: git-revision
+    value: pipelines-1.7
+  - name: IMAGE
+    value: image-registry.openshift-image-registry.svc:5000/$(context.pipelineRun.namespace)/pipelines-vote-ui
+  - name: insecure-skip-tls-verify
+    value: "true"
+  workspaces:
```

Comment:
Wondering if the best practice here would be to split those into two different workspace: one which holds the config the other one which holds the source; probably makes sense to split those IMO, WDYT?

Reply:
We've had a case of a tekton build pipeline which used two workspaces for different source elements, and where the build task that relied on the two workspaces did not get scheduled because the one workspace was relying on a PVC created on nodeA while the other workspace was relying on a PVC created on nodeB.
While I find the workspace separation good in principle, this might require more complex configuration to ensure that we don't end in a situation where some tasks in the pipeline cannot run because of how the configuration was applied in reality.

Reply:
Aight noted, I'd assume this one would be a review comment given by the Tekton maintainers as well what would be more preferable, looking at other tasks they do split things but we'll see.